Build/_build.csproj

```diff
@@ -22,8 +22,8 @@
     <PackageDownload Include="ReportGenerator" Version="[5.1.23]" />
     <PackageDownload Include="xunit.runner.console" Version="[2.5.0]" />
     <PackageDownload Include="Node.js.redist" Version="[16.20.0]" />
-    <PackageReference Include="LibGit2Sharp" Version="0.27.2" />
-    <PackageReference Include="Nuke.Common" Version="7.0.2" />
+    <PackageReference Include="LibGit2Sharp" Version="0.30.0" />
+    <PackageReference Include="Nuke.Common" Version="9.0.1" />
     <PackageReference Include="Nuke.Components" Version="7.0.2" />
     <PackageDownload Include="Yarn.MSBuild" Version="[1.22.19]" />
```

Comment:
Why is this dependency in again? This isn't needed anymore, since we use npm now

Reply:
This is `master`